sdk/identity/azure-identity/test/ut/azure_cli_credential_test.cpp

```diff
@@ -22,617 +22,602 @@ using Azure::Core::Credentials::TokenCredentialOptions;
 using Azure::Core::Credentials::TokenRequestContext;
 using Azure::Identity::AzureCliCredentialOptions;
 
-namespace {
-constexpr auto InfiniteCommand =
+namespace Azure { namespace Identity { namespace Test {
```

Comment:
Best to review this with whitespace off. All I am doing here is adding the `Test` nested namespace:
https://github.com/Azure/azure-sdk-for-cpp/pull/5331/files?diff=split&w=1